pkg/storaged/format-dialog.jsx

```diff
@@ -323,22 +323,14 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                           value: offer_keep_keys ? " keep" : "none",
                           visible: vals => vals.type != "dos-extended",
                           nested_fields: [
-                              PassInput("passphrase", _("Passphrase"),
-                                        {
-                                            validate: function (phrase, vals) {
-                                                if (vals.crypto != " keep" && phrase === "")
-                                                    return _("Passphrase cannot be empty");
-                                            },
-                                            visible: vals => is_encrypted(vals) && vals.crypto != " keep",
-                                        }),
-                              PassInput("passphrase2", _("Confirm"),
-                                        {
-                                            validate: function (phrase2, vals) {
-                                                if (vals.crypto != " keep" && phrase2 != vals.passphrase)
-                                                    return _("Passphrases do not match");
-                                            },
-                                            visible: vals => is_encrypted(vals) && vals.crypto != " keep",
-                                        }),
+                              NewPassInput("passphrase", _("Passphrase"),
+                                           {
+                                               validate: function (phrase, vals) {
+                                                   if (vals.crypto != " keep" && phrase === "")
+                                                       return _("Passphrase cannot be empty");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18906-20230613-094309-5e721175-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)